requirements-testing.txt

```diff
@@ -4,4 +4,3 @@ ndx-events>=0.2.0  # for special tests to ensure load_namespaces is set to allow
 parameterized>=0.8.1
 ndx-miniscope
 spikeinterface[qualitymetrics]>=0.99.1
```

Comment:
why do we still have spikeinterface though?

Reply:
Not SpikeInterface in particular, but the deps it needs for the `[qualitymetrics]` extra are used to test an advance feature of SI (sparisty I believe? Or something like that)